oqsprov/oqs_sig.c

```diff
@@ -1176,6 +1196,7 @@ static void oqs_sig_freectx(void *vpoqs_sigctx) {
     OPENSSL_free(ctx->aid);
     ctx->aid = NULL;
     ctx->aid_len = 0;
+    OPENSSL_free(ctx->context_string);
```

Comment:
```suggestion
    OPENSSL_free(ctx->context_string);
    ctx->context_string = NULL;
    ctx->context_string_length = 0;
```
Kind of a trivial point, but it's done for the other struct members, so may as well be consistent.

Reply:
I was briefly tempted to remove the other pretty pointless lines, but so what... See latest commit.